### PR TITLE
WebAPI: Update Method pages to modern structure (part 27)

### PR DESCRIPTION
**Pull request by OnkarRuikar**
### Summary
Updates `EmbedLiveSample` macros.
Increases height to avoid vertical scrollbar.

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/document/getelementbyid/index.md [hunk 1]
```diff
@@ -62,15 +62,15 @@ function changeColor(newColor) {
...
+{{ EmbedLiveSample('Examples', 250, 120) }}
 
 ## Usage notes
 
 The capitalization of `"Id"` in the name of this method _must_ be correct for the code to function; `getElementByID()` is _not_ valid and will not work, however natural it may seem.
 
 Unlike some other element-lookup methods such as {{domxref("Document.querySelector()")}} and {{domxref("Document.querySelectorAll()")}}, `getElementById()` is only available as a method of the global `document` object, and _not_ available as a method on all element objects in the DOM. Because ID values must be unique throughout the entire document, there is no need for "local" versions of the function.
 
-## Examples
+### Example
```
Comment (teoli2003) [inline files/en-us/web/api/document/getelementbyid/index.md:73]:
```suggestion
## Examples
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/document/getelementbyid/index.md (hunk 1 above)
Comment (OnkarRuikar) [reply files/en-us/web/api/document/getelementbyid/index.md:73]: This one is related to the `## Usage notes` section.
It demonstrates the fact that, "`getElementById()` is only available as a method of the global `document` object", explained in section above.
So, it needs to be a sub section with `###`.

Comment (teoli2003) [reply files/en-us/web/api/document/getelementbyid/index.md:73]: Yes, you are right. My bad.

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**